stdlib/timezone/timezone.flux

```diff
@@ -0,0 +1,7 @@
+package timezone
+
+
+utc = {zone: "UTC", offset: 0s}
+fixed = (offset) => ({zone: utc.zone, offset: offset})
+
+builtin location : (name: string) => {zone: string, offset: duration}
```

Comment:
These names work well. Can you add some comments to each value explaining their purpose etc?

The way I read the names are like this:

```
timezone.location(name: "XXX") // creates a timezone based on a specific location
timezone.fixed(offset: -4h) // creates a timezone that has a constant fixed offset from UTC
```

I like it.